frontend/src/components/Map/index.js

```diff
@@ -23,617 +32,640 @@ export default function Map() {
-          fips: "06095",
-          date: "2020-05-06",
-          cases: 342,
-        },
-        {
-          deaths: 4,
-          county: "Sonoma",
-          state: "California",
-          fips: "06097",
-          date: "2020-05-06",
-          cases: 286,
-        },
-        {
-          deaths: 17,
-          county: "Stanislaus",
-          state: "California",
-          fips: "06099",
-          date: "2020-05-06",
-          cases: 461,
-        },
-        {
-          deaths: 2,
-          county: "Sutter",
-          state: "California",
-          fips: "06101",
-          date: "2020-05-06",
-          cases: 31,
-        },
-        {
-          deaths: 1,
-          county: "Tehama",
-          state: "California",
-          fips: "06103",
-          date: "2020-05-06",
-          cases: 1,
-        },
-        {
-          deaths: 41,
-          county: "Tulare",
-          state: "California",
-          fips: "06107",
-          date: "2020-05-06",
-          cases: 916,
-        },
-        {
-          deaths: 0,
-          county: "Tuolumne",
-          state: "California",
-          fips: "06109",
-          date: "2020-05-06",
-          cases: 2,
-        },
-        {
-          deaths: 19,
-          county: "Ventura",
-          state: "California",
-          fips: "06111",
-          date: "2020-05-06",
-          cases: 608,
-        },
-        {
-          deaths: 20,
-          county: "Yolo",
-          state: "California",
-          fips: "06113",
-          date: "2020-05-06",
-          cases: 172,
-        },
-        {
-          deaths: 1,
-          county: "Yuba",
-          state: "California",
-          fips: "06115",
-          date: "2020-05-06",
-          cases: 19,
+  data.then(function (usData) {
+    // Add source for state polygons hosted on Mapbox, based on US Census Data:
+    // https://www.census.gov/geo/maps-data/data/cbf/cbf_state.html
+    map.addSource("states", {
+      type: "vector",
+      url: "mapbox://mapbox.us_census_states_2015",
+    });
+
+    // exclude states outside the 50 states
+    const expression = ["match", ["get", "STATE_ID"]];
+    usData.forEach(function (row) {
+      var stateID = row.state;
+      if (stateID in stateToFIPS) {
+        expression.push(stateToFIPS[stateID], get_color(row.group));
+      }
+    });
+    expression.push("rgba(0,0,0,0)"); // Last value is the default, used where there is no data
+
+    map.addLayer(
+      {
+        id: "states-join",
+        type: "fill",
+        source: "states",
+        minzoom: countryMinZoom,
+        // maxzoom: statesMinZoom,
+        "source-layer": "states",
+        paint: {
+          "fill-color": expression,
+          "fill-outline-color": fillOutlineColor,
         },
-      ];
+      },
+      "waterway-label"
+    );
+  });
+}
 
-      var values = countyData.map((county) => county.deaths); //get all the death data points
+function addUSCountyLayer(map) {
```

Comment:
The use of this method is commented out, can't we remove it?

Reply:
sure thing
We should keep the map tiles source somewhere at hand. Perhaps in the issue #86 ?

Reply:
Added a reference to the issue!